### PR TITLE
Require libz3-dev for ubuntu PPA builds.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: scripts/release_ppa.sh [hunk 1]
```diff
@@ -54,14 +54,24 @@ keyid=703F83D0
 email=builds@ethereum.org
 packagename=solc
 
-for distribution in trusty vivid xenial yakkety zesty
+for distribution in trusty vivid xenial zesty
```
Comment (axic) [inline scripts/release_ppa.sh:57]: yaketty was removed from the ppa?

File: scripts/release_ppa.sh [hunk 2]
```diff
@@ -54,14 +54,24 @@ keyid=703F83D0
...
 mkdir $distribution
 cd $distribution
 
+# Dependency
+if [ $distribution = trusty -o $distribution = vivid ]
+then
+    Z3DEPENDENCY=""
+else
+    Z3DEPENDENCY="libz3-dev,
+               "
```
Comment (axic) [inline scripts/release_ppa.sh:70]: This looks so weird but I guess the package format depends on it.

File: scripts/release_ppa.sh (hunk 1 above)
Comment (chriseth) [reply scripts/release_ppa.sh:57]: ubuntu does not accept any packages for yaketty anymore.

File: scripts/release_ppa.sh (hunk 2 above)
Comment (chriseth) [reply scripts/release_ppa.sh:70]: I'm not sure if it is required but wanted to keep it properly indented.

**Review by axic — APPROVED** [review]

**Pull request merged**